packages/controllers/src/snaps/SnapController.ts

```diff
@@ -911,13 +915,17 @@ export class SnapController extends BaseController<
    *
    * @param snapId - The snap id to check.
    * @param version - The version of the snap to check.
+   * @param sourceShaSum - The shasum of the snap's source code.
    * @returns Whether the version of the snap is blocked or not.
    */
   async isBlocked(
     snapId: ValidatedSnapId,
     version: SemVerVersion,
+    sourceShaSum: string,
```

Comment:
Can we correct to `shasum` everywhere in this PR?

Reply:
Updated.